lib/server.js

```diff
@@ -1590,10 +1106,44 @@ function paginationStream(opts, onRecord, done) {
     }
 }
 
+function getPlacementData(options) {
+    assert.object(options, 'options');
+    assert.object(options.log, 'options.log');
+    assert.object(options.dataDirector, 'options.dataDirector');
+
+    function _getPlacementData(rpc) {
+        var msgid = rpc.requestId();
+        var argv = rpc.argv();
+
+        if (invalidArgs(rpc, argv, GDP_ARGS_SCHEMA)) {
+            return;
+        }
+
+        dtrace['getplacementdata-start'].fire(function () {
```

Comment:
These DTrace probes don't seem very useful (nothing really happens between the "start" and "done" probes), but I understand us wanting them for completeness' sake. The same applies to the log messages.

Thinking about it, it is a bit useful to tell that this function is at least being called, it's just that having them separate is the part that's not totally useful. That they are separate just puts us in line with how the other probes work.

Seems fine to me. I just wanted to mention it in case you had any other thoughts here.

Reply:
I can definitely see that (ping @kellymclaughlin for his thoughts).  I vote to keep them separate to match how most dtrace probes are done, but I don't feel strongly about it at all.

Reply:
Yes, definitely kinda silly here, but just to keep uniformity with the rest of the code. This function in particular I expect will get expanded at some point because we will want to be able to conditionally pull the data if it has changed. So I expect at some point it will become less silly than it looks now.

Reply:
Agreed here. Seems fine to keep as-is for the sake of future proofing.